deb-package-builder/extensions/mongo/debian/copyright

```diff
@@ -0,0 +1,72 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: mongo
+Upstream-Contact: Daniel Holmes <dan@djcentric.com>
+Source: https://pecl.php.net/package/mongo
+
+Files: *
```

Comment:
According to https://pecl.php.net/package/mongo, it seems like the Apache License

Reply:
Source code shows Apache 2.0. Is there something that needs changing here?

Reply:
Below this line specifies the copyright holder and the license. It looks like the copyright holder should be `Copyright 2009-2014 MongoDB, Inc.` and the license should be `Apache-2.0`